### DISCUSSION TITLE
Add "Snooze" on Github Pull Request

### DISCUSSION THREAD

**Discussion by nafiesl** [community/community#17653] [Pull Requests]
Hi Github,

It would be nice if we have a **"snooze"** or **"remind me"** or **"postpone"** feature on GitHub Pull Request (and Issue). It is more like Gmail "snooze" (image below), where we can determine when we want to be reminded. This will be very useful if we get a bunch of pull requests to review or to follow up.

![screen_2022-06-02_004](https://user-images.githubusercontent.com/8721551/171573209-7028f7d9-c50c-4751-9f8e-643c1acefe83.jpg)

If possible, I would suggest these:

1. Remind ourselves (useful if we hold the work then get back to it later)
2. Remind the reviewers (useful if the reviewer is currently busy, we can remind them for the next **x hours**)

I am not sure if GitHub already has such a feature. But so far, I need to set the reminders somewhere else to remind me about the Pull Requests or Issues that I need to take a look later.

Thanks in advance!

**Comment by tgmpje** [comment]
I'm using [SnoozeThis](https://www.snoozethis.com/) for this. Simply add a comment to the PR:

`@SnoozeThis wait 2 days`

(or however long you want the PR to be snoozed)

**Reply by nafiesl** [reply]
Thanks for sharing the alternative tool @tgmpje. But it would be helpful if Github has this feature built in.

**Reply by tgmpje** [reply]
Totally agree
